File: MODUL 4/PRAK403-2410817210005-Muhammad_Irgi_Fahrezha.c
#include <stdio.h>

int main () {
    
    int bil1;
    int bil2;
    int i;
    int ii;

    scanf("%d %d", &bil1, &bil2);

    for ( i = bil1, ii = bil2; i <= bil2 && ii >= bil1; i++, ii--){
        printf("%d %d", i, ii);
        if ( i < bil2 || ii > bil1) {
            printf(" - ");
        }
    }

    for ( i = bil1, ii = bil2; i >= bil2 && ii <= bil1; i--, ii++){
        printf("%d %d", i, ii);
        if (i > bil2 || ii < bil1) {
            printf(" - ");
        }
    }
    return 0;
}

File: MODUL 4/PRAK403-2410817210005-Muhammad_Irgi_Fahrezha.py
bil1, bil2 = input().split()

bil1 = int(bil1)
bil2 = int(bil2)

if bil1 <= bil2:
    i, ii = bil1, bil2
    while i <= bil2 and ii >= bil1:
        print(f"{i} {ii}", end="")
        if i < bil2 or ii > bil1:
            print(" - ", end="")
        i += 1
        ii -= 1


else:
    i, ii = bil1, bil2
    while i >= bil2 and ii <= bil1:
        print(f"{i} {ii}", end="")
        if i > bil2 or ii < bil1:
            print(" - ", end="")
        i -= 1
        ii += 1